src/fr/invinciblecomics/src/eu/kanade/tachiyomi/extension/fr/invinciblecomics/InvincibleComics.kt

```diff
@@ -22,6 +22,20 @@ class InvincibleComics : HttpSource() {
 
     override val supportsLatest = true
 
+    override val client = network.cloudflareClient.newBuilder().addNetworkInterceptor { chain ->
```

Comment:
```suggestion
    override val client = network.cloudflareClient.newBuilder().addInterceptor { chain ->
```
You cannot send another request inside a network interceptor.